lib/r/source.dart

```diff
@@ -323,4 +323,14 @@ Future<void> rSource(BuildContext context, WidgetRef ref, String script) async {
   code = rStripComments(code);
 
   ref.read(ptyProvider).write(const Utf8Encoder().convert(code));
+
+  // Optionally, show a SnackBar when the script starts executing.
```

Comment:
Should this be showing the message when the script finished, not when it starts?